src/store/Controller.cc

```diff
@@ -534,6 +534,9 @@ Store::Controller::freeMemorySpace(const int bytesRequired)
     if (memoryCacheHasSpaceFor(pagesRequired))
         return;
 
+    if (mem_hdr::ReplPolicyIdleNodesCount < static_cast<size_t>(pagesRequired))
```

Comment:
Please use Less() for this comparison to avoid casting and clarify what is going on here:

```suggestion
    // do not free anything if freeing everything freeable would not be enough
    if (Less(mem_hdr::ReplPolicyIdleNodesCount, pagesRequired))
```